src/services/penaltyDocuments.js

```diff
@@ -871,4 +872,34 @@ export default class PenaltyDocument {
 			}
 		});
 	}
+
+	async streamDocuments(event, context, callback) {
+
+		let minOffset = 9999999999.999;
+		let count = 0;
+
+		event.Records.forEach((record) => {
+			const item = parse(record.dynamodb.NewImage);
+			if (item.Offset < minOffset) {
+				minOffset = item.Offset;
+			}
+			count += 1;
+		});
+
+		const params = this.apnsMessageParams(minOffset, count);
+
+		await this.sendSnsMessage(params);
```

Comment:
do we need to catch errors on this potentially?

Reply:
Yes good point. Will add error handling here.